llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -1193,6 +1213,8 @@ defm FNEG_H: F2_Support_Half<"neg", fneg>;
 
 defm FSQRT : F2<"sqrt.rn", fsqrt>;
 
+defm FEXP2_H: F2_Support_Half_2<"ex2.approx", fexp2>;
```

Comment:
Are the f16 and f16x2 variants needed? won't the Patterns in NVPTXIntrinsics.td already handle this?

Reply:
Thanks. Removed.